sv-SE.lproj/Menu.strings

```diff
@@ -0,0 +1,258 @@
+/* 
+  Menu.strings
+  Typora
+
+ by hi@typora.io
+*/
+
+/* Typora Menu */
+
+"About Typora" = "Om Typora";
+"Preferences" = "Inställningar";
+"Check for Updates" = "Sök efter uppdateringar";
+"Services" = "Tjänster";
+"Hide Typora" = "Göm Typora";
+"Hide Others" = "Göm andra"; /* Context? */
+"Show All" = "Visa alla"; /* Context? "Visa allt" may be a better translation depending on context */
```

Comment:
They are on macOS, follow the standard transalation is OK ->  https://github.com/martnst/localize-mainmenu/blob/master/languages/sv.json